bin/kafka-run-class.sh

```diff
@@ -243,6 +243,14 @@ fi
 (( WINDOWS_OS_FORMAT )) && LOG_DIR=$(cygpath --path --mixed "${LOG_DIR}")
 KAFKA_LOG4J_CMD_OPTS="-Dkafka.logs.dir=$LOG_DIR $KAFKA_LOG4J_OPTS"
 
+# slf4j provider settings
+if [ -z "$SLF4J_PROVIDER" ]; then
+  SLF4J_PROVIDER="org.slf4j.reload4j.Reload4jServiceProvider"
+fi
+
+# Add the slf4j provider to KAFKA_OPTS
+KAFKA_OPTS="$KAFKA_OPTS -Dslf4j.provider=$SLF4J_PROVIDER"
```

Comment:
Could we merge this new property into either `KAFKA_LOG4J_OPTS` or `KAFKA_LOG4J_CMD_OPTS`? Also, please add it to `kafka-run-class.bat` too

Reply:
Indeed, good ones. updated.